kernel/kernel-defaults/src/main/java/io/delta/kernel/defaults/internal/parquet/TimestampConverters.java

```diff
@@ -90,6 +90,12 @@ public static Converter createTimestampConverter(int initialBatchSize, Type type
     public static Converter createTimestampNtzConverter(int initialBatchSize, Type typeFromFile) {
```

Comment:
can this be merged into the `createTimestampConverter` method above? Looks like the code is exactly same except in couple of lines.

Reply:
Yup, that makes sense. I'll update.

Reply:
thanks Venki, merged them to a common method